validation/userRegisterValidator.ts

```diff
@@ -0,0 +1,14 @@
+import * as registrationValidator from "yup";
```

Comment:
I should be more clear with my comments. My previous comment suggested that this file be renamed to registrationValidator. What I meant was that the file --> ./validation/userRegisterValidator.ts be renamed to ./validation/registrationValidatory.ts
Yup was ok as it way. Sorry for the confusion

Reply:
I will change userRegisterValidator.ts to userValidator.ts because it will cover user registration and user login validation. I will push it soon.

Reply:
Can it cover the use case for both register and login? Register requires 3 fields: userName, email and password. Login only requires 2 fields: email and password.

Reply:
I've done it but I didn't push it. I will push it now.